app/src/main/java/eu/danman/zidostreamer/zidostreamer/BootReceiver.java

```diff
@@ -20,7 +20,7 @@ public void onReceive(Context context, Intent intent) {
         SharedPreferences settings = PreferenceManager.getDefaultSharedPreferences(context.getApplicationContext());
         Boolean start_after_boot = settings.getBoolean("start_after_boot", false);
 
-        Toast.makeText(context, "ALEEEEEEEEEEE!", Toast.LENGTH_LONG);
+        Toast.makeText(context, "ALEEEEEEEEEEE!", Toast.LENGTH_LONG).show();
```

Comment:
I didn't see the other guy's PR before, but I'm leaving this here because he didn't fix the show() here.